main.tf

```diff
@@ -0,0 +1,88 @@
+terraform {
+  required_version = ">= 1.0.0"
```

Comment:
Not exactly sure if this is the right versioning constraint long-term, but it seemed reasonable to only allow 1.0+ versions.